### PR TITLE
start with tiling endpoint

### PR DESCRIPTION
**Pull request by anikaweinmann**
Create endpint to generate grids.

Add endpoint to list all grids.

### CODE REVIEW

File: src/actinia_tiling_plugin/api/tiling/tiling_grid.py [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+__copyright__ = "Copyright 2022 mundialis GmbH & Co. KG"
+__maintainer__ = "mundialis GmbH % Co. KG"
+
+from flask import make_response, jsonify
+from flask_restful_swagger_2 import swagger
+import pickle
+from uuid import uuid4
+
+from actinia_core.rest.persistent_processing import PersistentProcessing
+from actinia_core.rest.resource_base import ResourceBase
```
Comment (mmacata) [inline src/actinia_tiling_plugin/api/tiling/tiling_grid.py]:
```suggestion
from actinia_core.processing.actinia_processing.ephemeral.persistent_processing \
     import PersistentProcessing
from actinia_core.rest.base.resource_base import ResourceBase
```

File: src/actinia_tiling_plugin/apidocs/tiling.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+            "schema": SimpleStatusCodeResponseModel,
+        }
+    },
+}
+
+# class
+grid_tiling_post_docs = {
+    # "summary" is taken from the description of the get method
+    "tags": ["Tiling"],
+    "description": "Creates grid tiles with a speciefied 'width' and 'height' "
```
Comment (neteler) [inline src/actinia_tiling_plugin/apidocs/tiling.py]:
```suggestion
    "description": "Creates grid tiles with a specified 'width' and 'height' "
```

File: docker/docker-compose.yml [hunk 3]
```diff
@@ -7,6 +7,7 @@ services:
       dockerfile: docker/Dockerfile
     volumes:
       - ..:/src/actinia-tiling-plugin/.
+      - /home/aweinmann/grassdata/:/actinia_core/grassdb/
```
Comment (mmacata) [inline docker/docker-compose.yml]: please make more generic

Comment (mmacata) [inline docker/docker-compose.yml]: please make more generic

File: src/actinia_tiling_plugin/resources/templating.py [hunk 4]
```diff
@@ -16,16 +16,17 @@
...
 
-Example core functionality
+Template loader file
 """
 
 __license__ = "GPLv3"
-__author__ = "Anika Weinmann"
-__copyright__ = "Copyright 2022 mundialis GmbH & Co. KG"
+__author__ = "Carmen Tawalika"
+__copyright__ = "Copyright 20108-2022 mundialis GmbH & Co. KG"
```
Comment (mmacata) [inline src/actinia_tiling_plugin/resources/templating.py:24]:
```suggestion
__copyright__ = "Copyright 2018-2022 mundialis GmbH & Co. KG"
```
```suggestion
__copyright__ = "Copyright 20108-2022 mundialis GmbH & Co. KG"
```

Comment (mmacata) [inline src/actinia_tiling_plugin/resources/templating.py:24]:
```suggestion
__copyright__ = "Copyright 2018-2022 mundialis GmbH & Co. KG"
```
```suggestion
__copyright__ = "Copyright 20108-2022 mundialis GmbH & Co. KG"
```

**Review by mmacata — APPROVED** [review]

**Pull request merged**